product-updates/knowledge-center/complete-resubmissions.html

```diff
@@ -294,7 +294,7 @@ <h1>Resubmitting Complete Data Files</h1>
         <p class="usa-intro">After data files are submitted, ACF checks the data for accuracy and completeness. If any errors are found in the data files submitted, ACF will notify you via email, with an error report containing a summary view of the issues. The following provides guidance on how to resubmit data files after the errors have been corrected.
         </p>
 <br>
-        <p>The new TANF Data Portal will not change reporting requirements related to which data needs to be reported. However, it will require a process adjustment to how some grantees currently submit revisions of their data—specifically that revisions to data already submitted to TDP for a given quarter will need to be submitted in full (i.e. complete data) rather than as partial data only containing cases that had been added or updated.</p>
+        <p>The new TANF Data Portal will not change reporting requirements related to which data needs to be reported. However, it will require a process adjustment to how some STTs (States, Tribes, and Territories) currently submit revisions of their data—specifically that revisions to data already submitted to TDP for a given quarter will need to be submitted in full (i.e. complete data) rather than as partial data only containing cases that had been added or updated.</p>
```

Comment:
@reitermb we decided on 3/20 to hold off on replacing `grantee` terminology for now. cc: @klinkoberstar